### PR TITLE
Upstart runner

### PR DESCRIPTION
**Pull request by galthaus**

### CODE REVIEW

File: script/runner [hunk 1]
```diff
@@ -172,6 +172,8 @@ service() {
             disable) chkconfig "$svc" off;;
             *)  command service "$svc" "$@";;
         esac
+    elif [ -x /sbin/initctl ] && /sbin/initctl version 2>/dev/null | grep -q upstart ; then
```
Comment (VictorLowther) [inline script/runner]: Try to not hardcode paths.  $PATH is sane here.

Comment (galthaus) [inline script/runner]: okay - so just initctl.

File: script/runner [hunk 2]
```diff
@@ -172,6 +172,8 @@ service() {
             disable) chkconfig "$svc" off;;
             *)  command service "$svc" "$@";;
         esac
+    elif [ -x initctl ] && initctl version 2>/dev/null | grep -q upstart ; then
```
Comment (galthaus) [inline script/runner]: Does -x do lookup in $PATH?

Comment (galthaus) [inline script/runner]: It doesn't so.  I guess I could do which initctl && ...

**Pull request merged**